__tests__/pages/kontak-darurat.test.tsx

```diff
@@ -0,0 +1,166 @@
+    render(<EmergencyContactSection emergency_contacts={emergency_contacts} />);
+    const description = screen.getByText(
+      /Kumpulan informasi situs dan kontak penting terkait fasilitas serta alat kesehatan untuk COVID-19 di level nasional./i,
+    );
+    expect(description).toBeVisible();
+    const faskes_title = screen.getByText(
+      /Cari kontak penyedia faskes secara detail perprovinsi se-Indonesia/i,
+    );
+    expect(faskes_title).toBeVisible();
+    const faskes_button = screen.getByText(/Telusuri Sekarang/i);
+    expect(faskes_button).toBeVisible();
+    const hotline_button = screen.getByText(/Hubungi 119 ext. 9/i);
+    expect(hotline_button).toBeVisible();
+    const hotline_title = screen.getByText(/Hotline Kementerian Kesehatan/i);
+    expect(hotline_title).toBeVisible();
+  });
+
+  it("renders the emergency contact card correctly", () => {
+    // eslint-disable-next-line no-lone-blocks
+    {
+      emergency_contacts.forEach((contact, i) => {
+        render(
+          <EmergencyContactCard
+            key={i}
+            description={contact.description}
+            image={contact.image}
+            name={contact.name}
+            url={contact.url}
+          />,
+        );
+        const cardImage = screen.getByAltText(
+          `kontak darurat covid ${contact.name}`,
+        );
+        expect(cardImage).toBeVisible();
+        const contact_name = screen.getByText(contact.name);
+        expect(contact_name).toBeVisible();
+        const contact_description = screen.getByText(contact.description);
+        expect(contact_description).toBeVisible();
+        const contact_button = screen.getByTestId(
+          `contact-button-${contact.name}`,
+        );
+        expect(contact_button).toBeVisible();
+        expect(contact_button).toHaveAttribute("href", contact.url);
+      });
+    }
+  });
+
+  it("renders the chatbot section correctly", () => {
+    render(<ChatbotSection />);
+    const chatbot_title = screen.getByText(/CovidAsha Chatbot 24x7/i);
+    expect(chatbot_title).toBeVisible();
+    const chatbot_description = screen.getByText(
+      /Layanan berbasis chatbot untuk mencari semua informasi terkait COVID-19/i,
+    );
+    expect(chatbot_description).toBeVisible();
+    const chatbot_button = screen.getByText(/Kirim chat sekarang/i);
+    expect(chatbot_button).toBeVisible();
+    expect(chatbot_button).toHaveAttribute(
+      "href",
+      "https://bit.ly/hotlinewarga",
+    );
+  });
+
+  it("renders the vaccine disclosure correctly", () => {
+    render(<VaccineSection vaccine_section={vaccine_section} />);
+    const disclosure_title = screen.getByText(/Mau Vaksin COVID-19?/i);
+    expect(disclosure_title).toBeVisible();
+  });
+
+  it("render the stacked vaccine links correctly", () => {
+    // eslint-disable-next-line no-lone-blocks
+    {
+      // eslint-disable-next-line array-callback-return
+      vaccine_section.map((link, i) => {
```

Comment:
Why do we need this ESLint disabling rule as well?
Tests are not supposed to return any value, so instead of `map`, you should have used `forEach`. But again, I would avoid doing loop in the tests by introducing the edge cases manually through `@jackfranklin/test-data-bot` builders.

Moreover, if you need to repeat a lot of cases, you can use `test.each()` instead.